CHANGES.rst

```diff
@@ -1,9 +1,12 @@
 0.16.2 (unreleased)
 ===================
 
+exposure_pipeline
+-----------------
+- Update source catalog file with tweaked coordinates. [#1373]
+
```

Comment:
```suggestion
```

now that #1375 is merged (switching change log handling to `towncrier`) this change log entry should be a file in `changes/` instead:
```shell
echo "Update source catalog file with tweaked coordinates." > changes/1373.general.rst
```

There currently isn't an `exposure_pipeline` change type but we can definitely add one to `pyproject.toml` if that's desired

(this is just because #1375 was merged while active PRs existed; new PRs will have these instructions in their checklist)

Reply:
Done. Thanks, @zacharyburnett!